types/redux-seamless-immutable/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for 0.4.0
+// Project: https://github.com/eadmundo/redux-seamless-immutable
+// Definitions by: Leandro Soares <https://github.com/SoaresMG>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.3
+
+declare module "redux-seamless-immutable" {
```

Comment:
Do not declare module here. Just take everything here outside the module declaration.

Reply:
Btw how does it work if there are no modules declared? I saw some packages doing it.

Reply:
Your `index.d.ts` is defining shape of `redux-seamless-immutable` module